src/routes/components/components.json

```
@@ -2301,5 +2301,14 @@
 		"addedOn": "2021-10-12",
 		"category": "SvelteKit Adapters",
 		"tags": ["integrations"]
+	},
+	{
+	    	"title": "svelte-adapter-appengine",
+	    	"url": "https://github.com/halfdanj/svelte-adapter-appengine",
+	    	"description": "SvelteKit adapter for Google Cloud App Engine",
+	    	"npm": "svelte-adapter-appengine",
+	    	"addedOn": "2021-10-20",
```

You need to include the stars or it won't render correctly
```suggestion
	    	"stars": 0,
	    	"addedOn": "2021-10-20",
```

---

Ah, I just copied the snippet from https://sveltesociety.dev/help/submitting?type=component, the snippet generator probably needs to be updated